### ISSUE TITLE
CI Checks failing on Pull Requests from forks

### ISSUE THREAD

**Issue by adamziel** [WordPress/blueprints#29]
Example: https://github.com/adamziel/blueprints/pull/28

![CleanShot 2024-04-19 at 17 18 28@2x](https://github.com/adamziel/blueprints/assets/205419/937fc06d-8ebc-48bc-bcd6-bc2642718b5a)

Errors:

```
RequestError [HttpError]: Resource not accessible by integration
    at /home/runner/work/_actions/actions/github-script/60a0d83039c74a4aee543508d2ffcb1c3799cdea/dist/index.js:9537:21
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async eval (eval at callAsyncFunction (/home/runner/work/_actions/actions/github-script/60a0d83039c74a4aee543508d2ffcb1c3799cdea/dist/index.js:35424:16), <anonymous>:56:3)
    at async main (/home/runner/work/_actions/actions/github-script/60a0d83039c74a4aee543508d2ffcb1c3799cdea/dist/index.js:35522:20) {
  status: 403,
  response: {
    url: 'https://api.github.com/repos/adamziel/blueprints/issues/28/comments',
```

This might be solved by switching to `pull_request_target` automation

and

```
Run python validate_pr.py
Error: blueprints/wpcli-markdown/blueprints/wpcli-markdown/blueprint.json file does not exist in this PR.
```

The latter one would be solved by rebasing, but we don't want to require everyone to rebase before merging.

**Comment by adamziel** [comment]
"Has a valid Blueprint.json" file is now fixed. The only remaining failure is for Pull Requests Comments

**Comment by adamziel** [comment]
Solved by [873f35f](https://github.com/adamziel/blueprints/commit/873f35fbecd05b5fe52ee65c7614a58f92ad382c)
